index.js

```diff
@@ -2,7 +2,8 @@ const express = require('express');
 const db_conn = require('./db_connection');
 // adding code here
 const bodyParser = require('body-parser');
-const userRoutes = require('./userRoutes')
+const userRoutes = require('./userRoutes');
+const surveyRoutes = require('./routes')
```

Comment:
Getting error on this line

Error: Cannot find module './routes'

I think because you are trying to import a folder instead of a specific file